Demo/EarlGreyExample/Podfile

```diff
@@ -39,7 +39,7 @@ target TEST_TARGET_SWIFT do
 end
 
 post_install do |installer|
-  load('configure_earlgrey_pods.rb')
```

Comment:
What am I missing?

<img width="911" alt="earlgrey_gem" src="https://cloud.githubusercontent.com/assets/1173057/15523035/37e06d7c-21e5-11e6-99a8-ae1ea2332774.png">

The `configure_earlgrey_pods.rb` file is gone. It's not there to load anymore. I deleted it because the code is now in the gem.